app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -4,9 +4,13 @@ subcategory: If you want to train to teach
 description: |-
   Find out how to train to teach in England as a student from outside the UK. Get help and guidance on your qualifications, funding and visa.
 related_content:
-    Get support training to teach if you're disabled: "/funding-and-support/if-youre-disabled"
-    Initial teacher training in England : "/train-to-be-a-teacher/initial-teacher-training"
+    Gain the equivalent of English QTS, from outside the UK : "/non-uk-teachers/international-qualified-teacher-status"
```

Comment:
Will this page act as the landing page for this section? If so, will the URL change? The URL says international student but the page says non-UK citizen

Reply:
We've deliberately not changed the URL yet - we wanted a quicker win of changing the title/h1 whilst we reviewed the outcomes of UR re international signposting

Would suggest that we leave the URL for now and review separately as this will involve redirects, a lot of inbound links and tracking!